README.md

```diff
@@ -110,9 +110,13 @@ To verify all the dependencies were successfully installed,
 try to run the tests with the following command:
 
 ```
-yarn test
+yarn test # headless tests
+yarn selenium-test # browser based tests
```

Comment:
what about calling this command `yarn browser-test`?
If we ever need to change the browser testing engine we won't need to rename the command.